gm4_blossoming_pots/data/gm4_blossoming_pots/function/flower/as_player.mcfunction

```diff
@@ -0,0 +1,14 @@
+# Sets up raycast, then redirects to as_marker, then revokes advancement
+# @s = player who interacted with an empty flower pot
+# at @s
+# run from advancement gm4_blossoming_pots:interact_flower_pot
+
+# Raycast for the pot (set up range and search)
+execute store result score @s gm4_blossoming_pots.range run attribute @s minecraft:block_interaction_range get 200
```

Comment:
Reading the interaction range for this is quite smart and should possibly become the new standard for the whole of Gamemode 4. We should discuss the performance impact of it internally.